lib/spear/connection.ex

```diff
@@ -83,24 +83,26 @@ defmodule Spear.Connection do
 
   @spec handle_responses(%__MODULE__{}, list()) :: %__MODULE__{}
   defp handle_responses(state, responses) do
-    Enum.reduce(responses, state, &process_response/2)
+    responses
+    |> Enum.reduce(state, &process_response/2)
+    |> Request.continue_requests()
```

Comment:
this gets invoked whenever the connection receives network traffic from the server

it triggers new attempts at all request streams in state (because a new message from the server may have included WINDOW_UPDATE frames)